src/components/Tokens/TokenDetails/ChartSection.tsx

```diff
@@ -17,13 +17,13 @@ import { L2NetworkLogo, LogoContainer } from '../TokenTable/TokenRow'
 import PriceChart from './PriceChart'
 import ShareButton from './ShareButton'
 
-export const ChartHeader = styled.div`
+export const ChartHeader = styled.div<{ gap?: boolean }>`
   width: 100%;
   display: flex;
   flex-direction: column;
   color: ${({ theme }) => theme.textPrimary};
   gap: 4px;
-  margin-bottom: 24px;
+  margin-bottom: ${({ gap }) => gap && '24px'};
```

Comment:
The vertical size of the Chart should be consistent, or the skeleton loader will not match what is actually loaded (this may be feedback for design).